site/docs/pages/token/token-row.mdx

```diff
@@ -143,106 +105,205 @@ Token with an amount
+  <span class="ock-tokenrow-left">
+    <img
+      data-testid="ockTokenImage_Image"
+      src="https://dynamic-assets.coinbase.com/dbb4b4983bde81309ddab83eb598358eb44375b930b94687ebe38bc22e52c3b2125258ffb8477a5ef22e33d6bd72e32a506c391caa13af64c00e46613c3e5806/asset_icons/4113b082d21cc5fab17fc8f2d19fb996165bcce635e6900f7fc2d57c4ef33ae9.png"
+      style="width: 32px; height: 32px; border-radius: 50%; overflow: hidden;"
     />
-  </div>
+    <span class="ock-tokenrow-body">
+      <span class="ock-tokenrow-name">Ethereum</span>
+    </span>
+  </span>
+  <span data-testid="ockTokenRow_Amount" class="ock-tokenrow-data"></span>
+</button>
+
+<button data-testid="ockTokenRow_Container" class="ock-tokenrow-button">
+  <span class="ock-tokenrow-left">
+    <span class="ock-tokenrow-body">
+      <span class="ock-tokenrow-name">Ethereum</span>
+      <span class="ock-tokenrow-symbol">ETH</span>
+    </span>
+  </span>
+  <span data-testid="ockTokenRow_Amount" class="ock-tokenrow-data"></span>
+</button>
+
+<button data-testid="ockTokenRow_Container" class="ock-tokenrow-button">
+  <span class="ock-tokenrow-left">
+    <span class="ock-tokenrow-body">
+      <span class="ock-tokenrow-name">Ethereum</span>
+    </span>
+  </span>
+  <span data-testid="ockTokenRow_Amount" class="ock-tokenrow-data"></span>
+</button>
+```
+
+:::
+
+<App>
+  <TokenRow
+    token={{
+      address: '0x1234',
+      chainId: 1,
+      decimals: 18,
+      image:
+        'https://dynamic-assets.coinbase.com/dbb4b4983bde81309ddab83eb598358eb44375b930b94687ebe38bc22e52c3b2125258ffb8477a5ef22e33d6bd72e32a506c391caa13af64c00e46613c3e5806/asset_icons/4113b082d21cc5fab17fc8f2d19fb996165bcce635e6900f7fc2d57c4ef33ae9.png',
+      name: 'Ethereum',
+      symbol: 'ETH',
+    }}
+    hideSymbol
+  />
+
+{' '}
+
+<TokenRow
+  token={{
+    address: '0x1234',
+    chainId: 1,
+    decimals: 18,
+    image:
+      'https://dynamic-assets.coinbase.com/dbb4b4983bde81309ddab83eb598358eb44375b930b94687ebe38bc22e52c3b2125258ffb8477a5ef22e33d6bd72e32a506c391caa13af64c00e46613c3e5806/asset_icons/4113b082d21cc5fab17fc8f2d19fb996165bcce635e6900f7fc2d57c4ef33ae9.png',
+    name: 'Ethereum',
+    symbol: 'ETH',
+  }}
+  hideImage
+/>
+
+  <TokenRow
+    token={{
+      address: '0x1234',
+      chainId: 1,
+      decimals: 18,
+      image:
+        'https://dynamic-assets.coinbase.com/dbb4b4983bde81309ddab83eb598358eb44375b930b94687ebe38bc22e52c3b2125258ffb8477a5ef22e33d6bd72e32a506c391caa13af64c00e46613c3e5806/asset_icons/4113b082d21cc5fab17fc8f2d19fb996165bcce635e6900f7fc2d57c4ef33ae9.png',
+      name: 'Ethereum',
+      symbol: 'ETH',
+    }}
+    hideSymbol
+    hideImage
+  />
 </App>
 
 ## Props
 
 [`TokenRowReact`](/token/types#tokenrowreact)
+
+## CSS
+
+```css
+.ock-tokenrow-button {
+  @apply flex h-16 w-full cursor-pointer items-center justify-between px-2 py-1;
+  background: #ffffff;
+
+  &:hover {
+    background: #cacbce;
+  }
+
```

Comment:
In general let's avoid have too many spaces when shocasing the CSS, so it's lets code to scroll.